src/AuthModule/Components/Login/Login.tsx

```diff
@@ -47,8 +48,20 @@ export default function Login() {
       );
       toast.success(response.data.message);
       localStorage.setItem("adminToken",response?.data?.data?.token); 
-      saveLoginData(); 
-      navigate("/Admin");
+      // saveLoginData(); 
+      const encodedToken:any=localStorage.getItem("adminToken");
+    const decodedToken:any=jwtDecode(encodedToken); 
+    let role=decodedToken?.role;
+    localStorage.setItem('loginData',JSON.stringify(decodedToken)); 
+    localStorage.setItem('userRole',JSON.stringify(decodedToken?.role)); 
+
+      if(role=='user'){
+        navigate("/");
+      }
+      else{
+        navigate("/Admin");
```

Comment:
this will work, but being consistent with URL name is a good practice. We can make it `admin` for consistency throughout our application.